lib/concurrent/executor/java_thread_pool_executor.rb

```diff
@@ -149,15 +149,6 @@ def remaining_capacity
         @max_queue == 0 ? -1 : @executor.getQueue.remainingCapacity
       end
 
-      # This method is deprecated and will be removed soon.
-      # This method is supost to return the threads status, but Java API doesn't
-      # provide a way to get the thread status. So we return an empty Array instead.
-      def status
```

Comment:
I had completely forgotten about this. Nice catch. It's definitely time to remove it.